OpenRA.Mods.Common/Traits/Render/SelectionDecorations.cs

```diff
@@ -33,17 +33,18 @@ public class SelectionDecorationsInfo : ITraitInfo, ISelectionDecorationsInfo
 
 		public readonly Color SelectionBoxColor = Color.White;
 
+		public readonly string Image = "pips";
+
 		public object Create(ActorInitializer init) { return new SelectionDecorations(init.Self, this); }
 
+		[Desc("Don't change the order of these sequences."), SequenceReference("Image")]
```

Comment:
Asking politely isn't a very good approach.  You can enforce this by adding a [LoadUsing] method that ignores any attempted yaml overrides and returns the default sequences.